### PR TITLE
Improve usage driver to send usage based on cache

### LINKED ISSUE #155: Improve usage driver to send usage based on cache
The usage driver must retrieve all ports and get the usage of each one.

### PR DESCRIPTION
**Pull request by paulobressan**
Close #155

### CODE REVIEW

**Review by gonzalezzfelipe — APPROVED** [review]
Should you update the bootstrap as well?

File: bootstrap/daemon/daemon.toml.tftpl [hunk 1]
```diff
@@ -9,13 +9,27 @@ mode = "${mode}"
...
 "sasl.password" = "${kafka_password}"
 
+[kafka_cache]
+"bootstrap.servers" = "${broker_urls}"
+"group.id"= "${consumer_cache_name}"
 "auto.offset.reset" = "earliest"
+"security.protocol" = "SASL_SSL"
+"sasl.mechanisms" = "SCRAM-SHA-256"
+"sasl.username" = "${kafka_username}"
+"sasl.password" = "${kafka_password}"
```
Comment (gonzalezzfelipe) [inline bootstrap/daemon/daemon.toml.tftpl:35]: Excellent

**Review by gonzalezzfelipe — APPROVED** [review]
👍

File: bootstrap/daemon/sts.tf [hunk 2]
```diff
@@ -39,13 +55,14 @@ resource "kubernetes_deployment_v1" "daemon" {
...
-            container_port = local.port
+          volume_mount {
+            name       = "cache"
+            mount_path = "/var/cache"
           }
 
           volume_mount {
             name       = "config"
-            mount_path = "/fabric"
+            mount_path = "/fabric/config"
```
Comment (gonzalezzfelipe) [inline bootstrap/daemon/sts.tf:65]: If we change this, we also have to change the `DAEMON_CONFIG` env var. I think this change is not necessary though
```suggestion
            mount_path = "/fabric"
```

File: bootstrap/daemon/daemon.toml.tftpl (hunk 1 above)
Comment (gonzalezzfelipe) [inline bootstrap/daemon/daemon.toml.tftpl:35]: I think we are missing a db path for the cache right? that should point to `/var/cache/{consumer_name}.db` or something like that

Comment (paulobressan) [reply bootstrap/daemon/daemon.toml.tftpl:35]: true, let me add, tks

File: bootstrap/daemon/sts.tf (hunk 2 above)
Comment (paulobressan) [reply bootstrap/daemon/sts.tf:65]: yes, good caught

**Review by gonzalezzfelipe — APPROVED** [review]

**Pull request merged**